File: app/src/main/java/com/roy/betterrecyleview/FeedRootRecyclerView.java
package com.roy.betterrecyleview;

import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;

/**
 * Created by roy on 2017/3/9.
 */

public class FeedRootRecyclerView extends BetterRecyclerView {
    public FeedRootRecyclerView(Context context) {
        this(context, null);
    }

    public FeedRootRecyclerView(Context context, @Nullable AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public FeedRootRecyclerView(Context context, @Nullable AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    @Override
    public void requestDisallowInterceptTouchEvent(boolean disallowIntercept) {
    /* do nothing */
    }
}

File: app/src/main/java/com/roy/betterrecyleview/FeedRootRecyclerView.kt
package com.roy.betterrecyleview

import android.content.Context
import android.util.AttributeSet

/**
 * Created by roy on 2017/3/9.
 */

class FeedRootRecyclerView @JvmOverloads constructor(context: Context, attrs: AttributeSet? = null, defStyle: Int = 0) : BetterRecyclerView(context, attrs, defStyle) {

    override fun requestDisallowInterceptTouchEvent(disallowIntercept: Boolean) {
        /* do nothing */
    }
}
